File: client/src/app/model/VM/reportVM.ts
import { Report } from '../report';
import { Vehicle } from '../vehicle';
import { User } from '../user';
import { WorkEvent } from '../workEvent';



export class ReportVM extends Report {
  public vehicle:Vehicle = new Vehicle();
  /**
   *
   */
  constructor() {
    super();
    this.workEvents = new Array<WorkEvent>();
    this.user = new User();
  }
}

File: TemplateMongo/TemplateMongo/Model/VM/ReportVM.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace TemplateMongo.Model.VM
{
    public class ReportVM :Report
    {
        public Vehicle vehicle { get; set; }
        public ReportVM()
        {
            this.user = new User();
            this.workEvents = new List<WorkEvent>();
        }
    }
}
